Feature/migrate blockchain connector to desmos

oriolcanades opened the pull request.

wistefan commented on it/it.iml:
```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
This file should be removed, it belongs to the ide.

wistefan commented on it/provider-values.yaml:
```diff
@@ -1,28 +1,31 @@
+    repository: quay.io/digitelts/dlt-adapter
     # -- tag of the image to be used
-    tag: latest
+    tag: ""
```
Why is this left empty? If you want to use the tag from the remote chart, you should leave out the whole tag field.

wistefan commented on it/provider-values.yaml:
```diff
@@ -1,28 +1,31 @@
   replicaCount: 1
   image:
-    repository: quay.io/wi_stefan/dlt-adapter
+    repository: quay.io/digitelts/dlt-adapter
```
Do we need the image config here? It would be better to just use the one configured in the chart. Else the test does not test the chart anymore. The previous one was just a work around, since the real chart did not yet exist.

wistefan commented:
Could you please provide a short summary of the involved changes, so that we can use them for release notes?

oriolcanades commented:
This release focuses on significant architectural enhancements and reliability improvements, primarily through the integration of the broker-adapter and blockchain-connector into a unified solution. Key highlights include:

- **Unified Architecture**: Merged the broker-adapter and blockchain-connector into a single, cohesive solution, streamlining integration and improving efficiency.
- **Configuration Simplification**: Refactored the blockchain-connector to simplify its configuration process, facilitating easier deployment and maintenance.
- **Fault Tolerance**: Implemented Fault Tolerance policies to enhance system resilience and ensure continuous operation under adverse conditions.
- **Testing**: Expanded our test suite with additional Unit and Integration Tests, significantly improving code reliability and facilitating future development.
- **Bug Fixes**: Addressed and resolved various issues and bugs, enhancing the system's stability and performance.

wistefan commented:
@oriolcanades Could you please resolve the open questions?
